### DISCUSSION TITLE
Can I auto unfold admonitions on search?

### DISCUSSION THREAD

**Discussion by aretsmarvin** [squidfunk/mkdocs-material#7224] [Q&A]
When an admonition appears in a search result, and an user clicks on the search result it goes to the page where the admonition is, but it doesn't unfold or highlight the admonition.
Can I make this working? Or is this not possible?

**Comment by kamilkrzyskow** [comment]
Hello @aretsmarvin,
assuming you're using the highlight feature that adds the `?h=search-query` parameter to the URL, this should be possible using additional JavaScript. You would `document.querySelectorAll("details")` and then check for the search query inside each detail and then you could open it.
- https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript
- https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText
- https://www.w3schools.com/jsref/prop_details_open.asp
